src/domain/platform/admin/templates/CommunityGuidelines/CommunityGuidelinesTemplateForm.tsx

```diff
@@ -35,6 +37,7 @@ interface CommunityGuidelinesTemplateFormProps {
   visual?: Visual;
   onSubmit: (values: CommunityGuidelinesTemplateFormSubmittedValues) => void;
   actions: ReactNode | ((formState: FormikProps<CommunityGuidelinesTemplateFormValues>) => ReactNode);
+  guidelinesTemplateId?: string;
```

Comment:
this one you'd pass if the form is loaded to edit the template, in this case, we are creating a template, so no need to pass it